config/ufs/config_files.xml

```diff
@@ -107,7 +107,7 @@
   <entry id="COMP_ROOT_DIR_ATM">
     <type>char</type>
     <values>
-      <value component="datm"      >$SRCROOT/components/cdeps/datm</value>
+      <value component="datm"      >$SRCROOT/src/model/CDEPS/datm</value>
```

Comment:
we are checking out CDEPS under components/cdeps for HAFs application. I think this needs to be changed and we need to checkout under src/model/CDEPS/ after this PR. Am I right?

Reply:
Yes, for s2s I added it as a submodule to ufs-s2s-model, you can do that or keep it in the Externals.cfg file
but you will need the change to manage_externals too.